supported-targets.md

```diff
@@ -79,7 +93,9 @@
 | Freescale         | K22F        | ✔ Full        | pre-v1.6           | kinetis.c      | -
 | Freescale         | K64F        | ✔ Full        | v1.6.2             | kinetis.c      | -
 | NXP               | S32K11      | ✔ Full        | v1.8.0             | kinetis.c      | -
-| NXP               | S32K14      | ✔ Full        | v1.8.0             | kinetis.c      | -
+| NXP               | S32K11      | ✔ Full        | v1.8.0             | kinetis.c      | -
+| NXP               | S32K344     | ✔ Full        | v2.0.0             | s32k3xx.c      | -
+| NXP               | i.MX RT6    | ✔ Full        | v1.10.0            | imxrt.c        | -
```

Comment:
`i.MX RT6` is duplicated on line 102 - 4 lines below. Ideally this would be putting them in numerical order (RT5, 6, 10, 11) but not duplicating.